doc/cephfs/createfs.rst

```diff
@@ -48,7 +48,36 @@ Once the pools are created, you may enable the file system using the ``fs new``
 
 .. code:: bash
 
-    $ ceph fs new <fs_name> <metadata> <data>
+    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover]
+
+This command creates a new file system. The file system name and metadata pool
+name are self-explanatory. The specified data pool is the default data pool and
+cannot be changed once set. Each file system has its own set of MDS daemons
```

Comment:
`The file system name and metadata pool name are self-explanatory.`  One might argue that if they are, this sentence is superfluous ;)